hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java

```diff
@@ -942,4 +945,66 @@ public void storeRootCACertificate(String pemEncodedCert, boolean force)
       lock.unlock();
     }
   }
+
+  @Override
+  public boolean processCrl(CRLInfo crl){
```

Comment:
Do we assume that the revocation time on the CRL has expired when this method is called?

Reply:
Yes. This will be used by the background service when the crl has passed the revocation time. It is not fully integrated yet and is intended to be used by the next patch.